.changeset/chilled-donuts-search.md

```diff
@@ -0,0 +1,8 @@
+---
+'@envelop/response-cache': minor
```

Comment:
@n1ru4l should be major?

Reply:
pre `1.x.x` a minor bump is actually considered a major as long as the package is unstable (see https://semver.org/#spec-item-4)

I am sure we will do a few more breaking changes to the response cache, before making it a full v1. We still need a bit more feedback from the community.

Reply:
Oh, i see, makes sense, thanks!